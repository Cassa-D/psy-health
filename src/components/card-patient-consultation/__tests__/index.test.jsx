import React from "react";
import renderer from "react-test-renderer";

import CardPatientConsultation from "../../card-patient-consultation";

describe("Render tests", () => {
  test("render card-patient-consultation with reschedule button", () => {
    const tree = renderer
      .create(
        <CardPatientConsultation
          constructDate={() => { }}
          constructHour={() => { }}
          cancelAppointment={() => { }}
          rescheduleAppointment={() => { }}
          onePsi={[
            {
              email: "mario@gmail.com",
              name: "Mario Silva",
              crp: "02/6667",
              price: 250,
              rating: "5",
              image: "img",
              id: 9,
            },
          ]}
          buttonOrAvaliation
        />
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });

  test("render card-patient-consultation with avaliation button", () => {
    const tree = renderer
      .create(
        <CardPatientConsultation
          constructDate={() => { }}
          constructHour={() => { }}
          cancelAppointment={() => { }}
          rescheduleAppointment={() => { }}
          onePsi={[
            {
              email: "joanacmota@gmail.com",
              name: "Joana Camargo Mota",
              crp: "01/74521",
              price: 90,
              rating: "4.5",
              image: "img",
              id: 1,
            },
          ]}
          buttonOrAvaliation={false}
        />
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
