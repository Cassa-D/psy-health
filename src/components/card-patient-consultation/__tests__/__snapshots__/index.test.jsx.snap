// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render tests render card-patient-consultation with avaliation button 1`] = `
<div
  className="sc-bdnylx fIhcHt"
>
  <div
    className="sc-gtssRu eDnBbj"
  >
    <div
      className="sc-dlnjPT fHtTbp"
    >
      <img
        className="sc-hKFyIo kRmnGk"
        src="img"
      />
    </div>
    <div
      className="sc-eCApGN BqijQ"
    >
      <div
        className="sc-gKAblj kmUQlX"
      >
        Joana Camargo Mota
      </div>
      <div
        className="sc-jSFkmK hOlCfU"
      >
        CRP: 01/74521
      </div>
      <ul
        className="ant-rate ant-rate-disabled sc-kEqYlL lhMFEB"
        onBlur={null}
        onFocus={null}
        onKeyDown={null}
        onMouseLeave={null}
        role="radiogroup"
        style={Object {}}
        tabIndex={-1}
      >
        <li
          className="ant-rate-star ant-rate-star-full"
        >
          <div
            aria-checked="true"
            aria-posinset={1}
            aria-setsize={5}
            onClick={null}
            onKeyDown={null}
            onMouseMove={null}
            role="radio"
            tabIndex={-1}
          >
            <div
              className="ant-rate-star-first"
            >
              <span
                aria-label="star"
                className="anticon anticon-star"
                role="img"
              >
                <svg
                  aria-hidden="true"
                  className=""
                  data-icon="star"
                  fill="currentColor"
                  focusable="false"
                  height="1em"
                  viewBox="64 64 896 896"
                  width="1em"
                >
                  <path
                    d="M908.1 353.1l-253.9-36.9L540.7 86.1c-3.1-6.3-8.2-11.4-14.5-14.5-15.8-7.8-35-1.3-42.9 14.5L369.8 316.2l-253.9 36.9c-7 1-13.4 4.3-18.3 9.3a32.05 32.05 0 00.6 45.3l183.7 179.1-43.4 252.9a31.95 31.95 0 0046.4 33.7L512 754l227.1 119.4c6.2 3.3 13.4 4.4 20.3 3.2 17.4-3 29.1-19.5 26.1-36.9l-43.4-252.9 183.7-179.1c5-4.9 8.3-11.3 9.3-18.3 2.7-17.5-9.5-33.7-27-36.3z"
                  />
                </svg>
              </span>
            </div>
            <div
              className="ant-rate-star-second"
            >
              <span
                aria-label="star"
                className="anticon anticon-star"
                role="img"
              >
                <svg
                  aria-hidden="true"
                  className=""
                  data-icon="star"
                  fill="currentColor"
                  focusable="false"
                  height="1em"
                  viewBox="64 64 896 896"
                  width="1em"
                >
                  <path
                    d="M908.1 353.1l-253.9-36.9L540.7 86.1c-3.1-6.3-8.2-11.4-14.5-14.5-15.8-7.8-35-1.3-42.9 14.5L369.8 316.2l-253.9 36.9c-7 1-13.4 4.3-18.3 9.3a32.05 32.05 0 00.6 45.3l183.7 179.1-43.4 252.9a31.95 31.95 0 0046.4 33.7L512 754l227.1 119.4c6.2 3.3 13.4 4.4 20.3 3.2 17.4-3 29.1-19.5 26.1-36.9l-43.4-252.9 183.7-179.1c5-4.9 8.3-11.3 9.3-18.3 2.7-17.5-9.5-33.7-27-36.3z"
                  />
                </svg>
              </span>
            </div>
          </div>
        </li>
        <li
          className="ant-rate-star ant-rate-star-full"
        >
          <div
            aria-checked="true"
            aria-posinset={2}
            aria-setsize={5}
            onClick={null}
            onKeyDown={null}
            onMouseMove={null}
            role="radio"
            tabIndex={-1}
          >
            <div
              className="ant-rate-star-first"
            >
              <span
                aria-label="star"
                className="anticon anticon-star"
                role="img"
              >
                <svg
                  aria-hidden="true"
                  className=""
                  data-icon="star"
                  fill="currentColor"
                  focusable="false"
                  height="1em"
                  viewBox="64 64 896 896"
                  width="1em"
                >
                  <path
                    d="M908.1 353.1l-253.9-36.9L540.7 86.1c-3.1-6.3-8.2-11.4-14.5-14.5-15.8-7.8-35-1.3-42.9 14.5L369.8 316.2l-253.9 36.9c-7 1-13.4 4.3-18.3 9.3a32.05 32.05 0 00.6 45.3l183.7 179.1-43.4 252.9a31.95 31.95 0 0046.4 33.7L512 754l227.1 119.4c6.2 3.3 13.4 4.4 20.3 3.2 17.4-3 29.1-19.5 26.1-36.9l-43.4-252.9 183.7-179.1c5-4.9 8.3-11.3 9.3-18.3 2.7-17.5-9.5-33.7-27-36.3z"
                  />
                </svg>
              </span>
            </div>
            <div
              className="ant-rate-star-second"
            >
              <span
                aria-label="star"
                className="anticon anticon-star"
                role="img"
              >
                <svg
                  aria-hidden="true"
                  className=""
                  data-icon="star"
                  fill="currentColor"
                  focusable="false"
                  height="1em"
                  viewBox="64 64 896 896"
                  width="1em"
                >
                  <path
                    d="M908.1 353.1l-253.9-36.9L540.7 86.1c-3.1-6.3-8.2-11.4-14.5-14.5-15.8-7.8-35-1.3-42.9 14.5L369.8 316.2l-253.9 36.9c-7 1-13.4 4.3-18.3 9.3a32.05 32.05 0 00.6 45.3l183.7 179.1-43.4 252.9a31.95 31.95 0 0046.4 33.7L512 754l227.1 119.4c6.2 3.3 13.4 4.4 20.3 3.2 17.4-3 29.1-19.5 26.1-36.9l-43.4-252.9 183.7-179.1c5-4.9 8.3-11.3 9.3-18.3 2.7-17.5-9.5-33.7-27-36.3z"
                  />
                </svg>
              </span>
            </div>
          </div>
        </li>
        <li
          className="ant-rate-star ant-rate-star-full"
        >
          <div
            aria-checked="true"
            aria-posinset={3}
            aria-setsize={5}
            onClick={null}
            onKeyDown={null}
            onMouseMove={null}
            role="radio"
            tabIndex={-1}
          >
            <div
              className="ant-rate-star-first"
            >
              <span
                aria-label="star"
                className="anticon anticon-star"
                role="img"
              >
                <svg
                  aria-hidden="true"
                  className=""
                  data-icon="star"
                  fill="currentColor"
                  focusable="false"
                  height="1em"
                  viewBox="64 64 896 896"
                  width="1em"
                >
                  <path
                    d="M908.1 353.1l-253.9-36.9L540.7 86.1c-3.1-6.3-8.2-11.4-14.5-14.5-15.8-7.8-35-1.3-42.9 14.5L369.8 316.2l-253.9 36.9c-7 1-13.4 4.3-18.3 9.3a32.05 32.05 0 00.6 45.3l183.7 179.1-43.4 252.9a31.95 31.95 0 0046.4 33.7L512 754l227.1 119.4c6.2 3.3 13.4 4.4 20.3 3.2 17.4-3 29.1-19.5 26.1-36.9l-43.4-252.9 183.7-179.1c5-4.9 8.3-11.3 9.3-18.3 2.7-17.5-9.5-33.7-27-36.3z"
                  />
                </svg>
              </span>
            </div>
            <div
              className="ant-rate-star-second"
            >
              <span
                aria-label="star"
                className="anticon anticon-star"
                role="img"
              >
                <svg
                  aria-hidden="true"
                  className=""
                  data-icon="star"
                  fill="currentColor"
                  focusable="false"
                  height="1em"
                  viewBox="64 64 896 896"
                  width="1em"
                >
                  <path
                    d="M908.1 353.1l-253.9-36.9L540.7 86.1c-3.1-6.3-8.2-11.4-14.5-14.5-15.8-7.8-35-1.3-42.9 14.5L369.8 316.2l-253.9 36.9c-7 1-13.4 4.3-18.3 9.3a32.05 32.05 0 00.6 45.3l183.7 179.1-43.4 252.9a31.95 31.95 0 0046.4 33.7L512 754l227.1 119.4c6.2 3.3 13.4 4.4 20.3 3.2 17.4-3 29.1-19.5 26.1-36.9l-43.4-252.9 183.7-179.1c5-4.9 8.3-11.3 9.3-18.3 2.7-17.5-9.5-33.7-27-36.3z"
                  />
                </svg>
              </span>
            </div>
          </div>
        </li>
        <li
          className="ant-rate-star ant-rate-star-full"
        >
          <div
            aria-checked="true"
            aria-posinset={4}
            aria-setsize={5}
            onClick={null}
            onKeyDown={null}
            onMouseMove={null}
            role="radio"
            tabIndex={-1}
          >
            <div
              className="ant-rate-star-first"
            >
              <span
                aria-label="star"
                className="anticon anticon-star"
                role="img"
              >
                <svg
                  aria-hidden="true"
                  className=""
                  data-icon="star"
                  fill="currentColor"
                  focusable="false"
                  height="1em"
                  viewBox="64 64 896 896"
                  width="1em"
                >
                  <path
                    d="M908.1 353.1l-253.9-36.9L540.7 86.1c-3.1-6.3-8.2-11.4-14.5-14.5-15.8-7.8-35-1.3-42.9 14.5L369.8 316.2l-253.9 36.9c-7 1-13.4 4.3-18.3 9.3a32.05 32.05 0 00.6 45.3l183.7 179.1-43.4 252.9a31.95 31.95 0 0046.4 33.7L512 754l227.1 119.4c6.2 3.3 13.4 4.4 20.3 3.2 17.4-3 29.1-19.5 26.1-36.9l-43.4-252.9 183.7-179.1c5-4.9 8.3-11.3 9.3-18.3 2.7-17.5-9.5-33.7-27-36.3z"
                  />
                </svg>
              </span>
            </div>
            <div
              className="ant-rate-star-second"
            >
              <span
                aria-label="star"
                className="anticon anticon-star"
                role="img"
              >
                <svg
                  aria-hidden="true"
                  className=""
                  data-icon="star"
                  fill="currentColor"
                  focusable="false"
                  height="1em"
                  viewBox="64 64 896 896"
                  width="1em"
                >
                  <path
                    d="M908.1 353.1l-253.9-36.9L540.7 86.1c-3.1-6.3-8.2-11.4-14.5-14.5-15.8-7.8-35-1.3-42.9 14.5L369.8 316.2l-253.9 36.9c-7 1-13.4 4.3-18.3 9.3a32.05 32.05 0 00.6 45.3l183.7 179.1-43.4 252.9a31.95 31.95 0 0046.4 33.7L512 754l227.1 119.4c6.2 3.3 13.4 4.4 20.3 3.2 17.4-3 29.1-19.5 26.1-36.9l-43.4-252.9 183.7-179.1c5-4.9 8.3-11.3 9.3-18.3 2.7-17.5-9.5-33.7-27-36.3z"
                  />
                </svg>
              </span>
            </div>
          </div>
        </li>
        <li
          className="ant-rate-star ant-rate-star-zero"
        >
          <div
            aria-checked="true"
            aria-posinset={5}
            aria-setsize={5}
            onClick={null}
            onKeyDown={null}
            onMouseMove={null}
            role="radio"
            tabIndex={-1}
          >
            <div
              className="ant-rate-star-first"
            >
              <span
                aria-label="star"
                className="anticon anticon-star"
                role="img"
              >
                <svg
                  aria-hidden="true"
                  className=""
                  data-icon="star"
                  fill="currentColor"
                  focusable="false"
                  height="1em"
                  viewBox="64 64 896 896"
                  width="1em"
                >
                  <path
                    d="M908.1 353.1l-253.9-36.9L540.7 86.1c-3.1-6.3-8.2-11.4-14.5-14.5-15.8-7.8-35-1.3-42.9 14.5L369.8 316.2l-253.9 36.9c-7 1-13.4 4.3-18.3 9.3a32.05 32.05 0 00.6 45.3l183.7 179.1-43.4 252.9a31.95 31.95 0 0046.4 33.7L512 754l227.1 119.4c6.2 3.3 13.4 4.4 20.3 3.2 17.4-3 29.1-19.5 26.1-36.9l-43.4-252.9 183.7-179.1c5-4.9 8.3-11.3 9.3-18.3 2.7-17.5-9.5-33.7-27-36.3z"
                  />
                </svg>
              </span>
            </div>
            <div
              className="ant-rate-star-second"
            >
              <span
                aria-label="star"
                className="anticon anticon-star"
                role="img"
              >
                <svg
                  aria-hidden="true"
                  className=""
                  data-icon="star"
                  fill="currentColor"
                  focusable="false"
                  height="1em"
                  viewBox="64 64 896 896"
                  width="1em"
                >
                  <path
                    d="M908.1 353.1l-253.9-36.9L540.7 86.1c-3.1-6.3-8.2-11.4-14.5-14.5-15.8-7.8-35-1.3-42.9 14.5L369.8 316.2l-253.9 36.9c-7 1-13.4 4.3-18.3 9.3a32.05 32.05 0 00.6 45.3l183.7 179.1-43.4 252.9a31.95 31.95 0 0046.4 33.7L512 754l227.1 119.4c6.2 3.3 13.4 4.4 20.3 3.2 17.4-3 29.1-19.5 26.1-36.9l-43.4-252.9 183.7-179.1c5-4.9 8.3-11.3 9.3-18.3 2.7-17.5-9.5-33.7-27-36.3z"
                  />
                </svg>
              </span>
            </div>
          </div>
        </li>
      </ul>
    </div>
  </div>
  <div
    className="sc-iCoHVE ctpNfe"
  >
    <div
      className="sc-fujyUd gHpiOx"
    >
      <div
        className="sc-gKAblj kmUQlX"
      >
        Data
      </div>
      <div
        className="sc-pNWxx TxRCZ"
      />
    </div>
    <div
      className="sc-fujyUd gHpiOx"
    >
      <div
        className="sc-gKAblj kmUQlX"
      >
        Horário
      </div>
      <div
        className="sc-pNWxx TxRCZ"
      />
    </div>
    <div
      className="sc-fujyUd gHpiOx"
    >
      <div
        className="sc-gKAblj kmUQlX"
      >
        Valor
      </div>
      <div
        className="sc-pNWxx TxRCZ"
      >
        R$ 90,00
      </div>
    </div>
  </div>
  <div
    className="sc-jrsJCI juUtyQ"
  >
    <div
      className="sc-iqAbSa bVZoZK"
    >
      <button
        className="ant-btn sc-bqGHjH gQYdhN"
        onClick={[Function]}
        type="button"
      >
        <span>
          Avaliar
        </span>
      </button>
    </div>
  </div>
</div>
`;

exports[`Render tests render card-patient-consultation with reschedule button 1`] = `
<div
  className="sc-bdnylx fIhcHt"
>
  <div
    className="sc-gtssRu eDnBbj"
  >
    <div
      className="sc-dlnjPT fHtTbp"
    >
      <img
        className="sc-hKFyIo kRmnGk"
        src="img"
      />
    </div>
    <div
      className="sc-eCApGN BqijQ"
    >
      <div
        className="sc-gKAblj kmUQlX"
      >
        Mario Silva
      </div>
      <div
        className="sc-jSFkmK hOlCfU"
      >
        CRP: 02/6667
      </div>
      <ul
        className="ant-rate ant-rate-disabled sc-kEqYlL lhMFEB"
        onBlur={null}
        onFocus={null}
        onKeyDown={null}
        onMouseLeave={null}
        role="radiogroup"
        style={Object {}}
        tabIndex={-1}
      >
        <li
          className="ant-rate-star ant-rate-star-full"
        >
          <div
            aria-checked="true"
            aria-posinset={1}
            aria-setsize={5}
            onClick={null}
            onKeyDown={null}
            onMouseMove={null}
            role="radio"
            tabIndex={-1}
          >
            <div
              className="ant-rate-star-first"
            >
              <span
                aria-label="star"
                className="anticon anticon-star"
                role="img"
              >
                <svg
                  aria-hidden="true"
                  className=""
                  data-icon="star"
                  fill="currentColor"
                  focusable="false"
                  height="1em"
                  viewBox="64 64 896 896"
                  width="1em"
                >
                  <path
                    d="M908.1 353.1l-253.9-36.9L540.7 86.1c-3.1-6.3-8.2-11.4-14.5-14.5-15.8-7.8-35-1.3-42.9 14.5L369.8 316.2l-253.9 36.9c-7 1-13.4 4.3-18.3 9.3a32.05 32.05 0 00.6 45.3l183.7 179.1-43.4 252.9a31.95 31.95 0 0046.4 33.7L512 754l227.1 119.4c6.2 3.3 13.4 4.4 20.3 3.2 17.4-3 29.1-19.5 26.1-36.9l-43.4-252.9 183.7-179.1c5-4.9 8.3-11.3 9.3-18.3 2.7-17.5-9.5-33.7-27-36.3z"
                  />
                </svg>
              </span>
            </div>
            <div
              className="ant-rate-star-second"
            >
              <span
                aria-label="star"
                className="anticon anticon-star"
                role="img"
              >
                <svg
                  aria-hidden="true"
                  className=""
                  data-icon="star"
                  fill="currentColor"
                  focusable="false"
                  height="1em"
                  viewBox="64 64 896 896"
                  width="1em"
                >
                  <path
                    d="M908.1 353.1l-253.9-36.9L540.7 86.1c-3.1-6.3-8.2-11.4-14.5-14.5-15.8-7.8-35-1.3-42.9 14.5L369.8 316.2l-253.9 36.9c-7 1-13.4 4.3-18.3 9.3a32.05 32.05 0 00.6 45.3l183.7 179.1-43.4 252.9a31.95 31.95 0 0046.4 33.7L512 754l227.1 119.4c6.2 3.3 13.4 4.4 20.3 3.2 17.4-3 29.1-19.5 26.1-36.9l-43.4-252.9 183.7-179.1c5-4.9 8.3-11.3 9.3-18.3 2.7-17.5-9.5-33.7-27-36.3z"
                  />
                </svg>
              </span>
            </div>
          </div>
        </li>
        <li
          className="ant-rate-star ant-rate-star-full"
        >
          <div
            aria-checked="true"
            aria-posinset={2}
            aria-setsize={5}
            onClick={null}
            onKeyDown={null}
            onMouseMove={null}
            role="radio"
            tabIndex={-1}
          >
            <div
              className="ant-rate-star-first"
            >
              <span
                aria-label="star"
                className="anticon anticon-star"
                role="img"
              >
                <svg
                  aria-hidden="true"
                  className=""
                  data-icon="star"
                  fill="currentColor"
                  focusable="false"
                  height="1em"
                  viewBox="64 64 896 896"
                  width="1em"
                >
                  <path
                    d="M908.1 353.1l-253.9-36.9L540.7 86.1c-3.1-6.3-8.2-11.4-14.5-14.5-15.8-7.8-35-1.3-42.9 14.5L369.8 316.2l-253.9 36.9c-7 1-13.4 4.3-18.3 9.3a32.05 32.05 0 00.6 45.3l183.7 179.1-43.4 252.9a31.95 31.95 0 0046.4 33.7L512 754l227.1 119.4c6.2 3.3 13.4 4.4 20.3 3.2 17.4-3 29.1-19.5 26.1-36.9l-43.4-252.9 183.7-179.1c5-4.9 8.3-11.3 9.3-18.3 2.7-17.5-9.5-33.7-27-36.3z"
                  />
                </svg>
              </span>
            </div>
            <div
              className="ant-rate-star-second"
            >
              <span
                aria-label="star"
                className="anticon anticon-star"
                role="img"
              >
                <svg
                  aria-hidden="true"
                  className=""
                  data-icon="star"
                  fill="currentColor"
                  focusable="false"
                  height="1em"
                  viewBox="64 64 896 896"
                  width="1em"
                >
                  <path
                    d="M908.1 353.1l-253.9-36.9L540.7 86.1c-3.1-6.3-8.2-11.4-14.5-14.5-15.8-7.8-35-1.3-42.9 14.5L369.8 316.2l-253.9 36.9c-7 1-13.4 4.3-18.3 9.3a32.05 32.05 0 00.6 45.3l183.7 179.1-43.4 252.9a31.95 31.95 0 0046.4 33.7L512 754l227.1 119.4c6.2 3.3 13.4 4.4 20.3 3.2 17.4-3 29.1-19.5 26.1-36.9l-43.4-252.9 183.7-179.1c5-4.9 8.3-11.3 9.3-18.3 2.7-17.5-9.5-33.7-27-36.3z"
                  />
                </svg>
              </span>
            </div>
          </div>
        </li>
        <li
          className="ant-rate-star ant-rate-star-full"
        >
          <div
            aria-checked="true"
            aria-posinset={3}
            aria-setsize={5}
            onClick={null}
            onKeyDown={null}
            onMouseMove={null}
            role="radio"
            tabIndex={-1}
          >
            <div
              className="ant-rate-star-first"
            >
              <span
                aria-label="star"
                className="anticon anticon-star"
                role="img"
              >
                <svg
                  aria-hidden="true"
                  className=""
                  data-icon="star"
                  fill="currentColor"
                  focusable="false"
                  height="1em"
                  viewBox="64 64 896 896"
                  width="1em"
                >
                  <path
                    d="M908.1 353.1l-253.9-36.9L540.7 86.1c-3.1-6.3-8.2-11.4-14.5-14.5-15.8-7.8-35-1.3-42.9 14.5L369.8 316.2l-253.9 36.9c-7 1-13.4 4.3-18.3 9.3a32.05 32.05 0 00.6 45.3l183.7 179.1-43.4 252.9a31.95 31.95 0 0046.4 33.7L512 754l227.1 119.4c6.2 3.3 13.4 4.4 20.3 3.2 17.4-3 29.1-19.5 26.1-36.9l-43.4-252.9 183.7-179.1c5-4.9 8.3-11.3 9.3-18.3 2.7-17.5-9.5-33.7-27-36.3z"
                  />
                </svg>
              </span>
            </div>
            <div
              className="ant-rate-star-second"
            >
              <span
                aria-label="star"
                className="anticon anticon-star"
                role="img"
              >
                <svg
                  aria-hidden="true"
                  className=""
                  data-icon="star"
                  fill="currentColor"
                  focusable="false"
                  height="1em"
                  viewBox="64 64 896 896"
                  width="1em"
                >
                  <path
                    d="M908.1 353.1l-253.9-36.9L540.7 86.1c-3.1-6.3-8.2-11.4-14.5-14.5-15.8-7.8-35-1.3-42.9 14.5L369.8 316.2l-253.9 36.9c-7 1-13.4 4.3-18.3 9.3a32.05 32.05 0 00.6 45.3l183.7 179.1-43.4 252.9a31.95 31.95 0 0046.4 33.7L512 754l227.1 119.4c6.2 3.3 13.4 4.4 20.3 3.2 17.4-3 29.1-19.5 26.1-36.9l-43.4-252.9 183.7-179.1c5-4.9 8.3-11.3 9.3-18.3 2.7-17.5-9.5-33.7-27-36.3z"
                  />
                </svg>
              </span>
            </div>
          </div>
        </li>
        <li
          className="ant-rate-star ant-rate-star-full"
        >
          <div
            aria-checked="true"
            aria-posinset={4}
            aria-setsize={5}
            onClick={null}
            onKeyDown={null}
            onMouseMove={null}
            role="radio"
            tabIndex={-1}
          >
            <div
              className="ant-rate-star-first"
            >
              <span
                aria-label="star"
                className="anticon anticon-star"
                role="img"
              >
                <svg
                  aria-hidden="true"
                  className=""
                  data-icon="star"
                  fill="currentColor"
                  focusable="false"
                  height="1em"
                  viewBox="64 64 896 896"
                  width="1em"
                >
                  <path
                    d="M908.1 353.1l-253.9-36.9L540.7 86.1c-3.1-6.3-8.2-11.4-14.5-14.5-15.8-7.8-35-1.3-42.9 14.5L369.8 316.2l-253.9 36.9c-7 1-13.4 4.3-18.3 9.3a32.05 32.05 0 00.6 45.3l183.7 179.1-43.4 252.9a31.95 31.95 0 0046.4 33.7L512 754l227.1 119.4c6.2 3.3 13.4 4.4 20.3 3.2 17.4-3 29.1-19.5 26.1-36.9l-43.4-252.9 183.7-179.1c5-4.9 8.3-11.3 9.3-18.3 2.7-17.5-9.5-33.7-27-36.3z"
                  />
                </svg>
              </span>
            </div>
            <div
              className="ant-rate-star-second"
            >
              <span
                aria-label="star"
                className="anticon anticon-star"
                role="img"
              >
                <svg
                  aria-hidden="true"
                  className=""
                  data-icon="star"
                  fill="currentColor"
                  focusable="false"
                  height="1em"
                  viewBox="64 64 896 896"
                  width="1em"
                >
                  <path
                    d="M908.1 353.1l-253.9-36.9L540.7 86.1c-3.1-6.3-8.2-11.4-14.5-14.5-15.8-7.8-35-1.3-42.9 14.5L369.8 316.2l-253.9 36.9c-7 1-13.4 4.3-18.3 9.3a32.05 32.05 0 00.6 45.3l183.7 179.1-43.4 252.9a31.95 31.95 0 0046.4 33.7L512 754l227.1 119.4c6.2 3.3 13.4 4.4 20.3 3.2 17.4-3 29.1-19.5 26.1-36.9l-43.4-252.9 183.7-179.1c5-4.9 8.3-11.3 9.3-18.3 2.7-17.5-9.5-33.7-27-36.3z"
                  />
                </svg>
              </span>
            </div>
          </div>
        </li>
        <li
          className="ant-rate-star ant-rate-star-full"
        >
          <div
            aria-checked="true"
            aria-posinset={5}
            aria-setsize={5}
            onClick={null}
            onKeyDown={null}
            onMouseMove={null}
            role="radio"
            tabIndex={-1}
          >
            <div
              className="ant-rate-star-first"
            >
              <span
                aria-label="star"
                className="anticon anticon-star"
                role="img"
              >
                <svg
                  aria-hidden="true"
                  className=""
                  data-icon="star"
                  fill="currentColor"
                  focusable="false"
                  height="1em"
                  viewBox="64 64 896 896"
                  width="1em"
                >
                  <path
                    d="M908.1 353.1l-253.9-36.9L540.7 86.1c-3.1-6.3-8.2-11.4-14.5-14.5-15.8-7.8-35-1.3-42.9 14.5L369.8 316.2l-253.9 36.9c-7 1-13.4 4.3-18.3 9.3a32.05 32.05 0 00.6 45.3l183.7 179.1-43.4 252.9a31.95 31.95 0 0046.4 33.7L512 754l227.1 119.4c6.2 3.3 13.4 4.4 20.3 3.2 17.4-3 29.1-19.5 26.1-36.9l-43.4-252.9 183.7-179.1c5-4.9 8.3-11.3 9.3-18.3 2.7-17.5-9.5-33.7-27-36.3z"
                  />
                </svg>
              </span>
            </div>
            <div
              className="ant-rate-star-second"
            >
              <span
                aria-label="star"
                className="anticon anticon-star"
                role="img"
              >
                <svg
                  aria-hidden="true"
                  className=""
                  data-icon="star"
                  fill="currentColor"
                  focusable="false"
                  height="1em"
                  viewBox="64 64 896 896"
                  width="1em"
                >
                  <path
                    d="M908.1 353.1l-253.9-36.9L540.7 86.1c-3.1-6.3-8.2-11.4-14.5-14.5-15.8-7.8-35-1.3-42.9 14.5L369.8 316.2l-253.9 36.9c-7 1-13.4 4.3-18.3 9.3a32.05 32.05 0 00.6 45.3l183.7 179.1-43.4 252.9a31.95 31.95 0 0046.4 33.7L512 754l227.1 119.4c6.2 3.3 13.4 4.4 20.3 3.2 17.4-3 29.1-19.5 26.1-36.9l-43.4-252.9 183.7-179.1c5-4.9 8.3-11.3 9.3-18.3 2.7-17.5-9.5-33.7-27-36.3z"
                  />
                </svg>
              </span>
            </div>
          </div>
        </li>
      </ul>
    </div>
  </div>
  <div
    className="sc-iCoHVE ctpNfe"
  >
    <div
      className="sc-fujyUd gHpiOx"
    >
      <div
        className="sc-gKAblj kmUQlX"
      >
        Data
      </div>
      <div
        className="sc-pNWxx TxRCZ"
      />
    </div>
    <div
      className="sc-fujyUd gHpiOx"
    >
      <div
        className="sc-gKAblj kmUQlX"
      >
        Horário
      </div>
      <div
        className="sc-pNWxx TxRCZ"
      />
    </div>
    <div
      className="sc-fujyUd gHpiOx"
    >
      <div
        className="sc-gKAblj kmUQlX"
      >
        Valor
      </div>
      <div
        className="sc-pNWxx TxRCZ"
      >
        R$ 250,00
      </div>
    </div>
  </div>
  <div
    className="sc-jrsJCI juUtyQ"
  >
    <div
      className="sc-iqAbSa bVZoZK"
    >
      <button
        className="ant-btn sc-dIsAE eyiywg"
        onClick={[Function]}
        type="button"
      >
        <span>
          Remarcar
        </span>
      </button>
      <button
        className="ant-btn sc-crzoUp etlZPJ"
        onClick={[Function]}
        type="button"
      >
        <span>
          Cancelar
        </span>
      </button>
    </div>
  </div>
</div>
`;
